packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMarkerUtilities.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <GoogleMaps/GoogleMaps.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GoogleMarkerUtilities : NSObject
```

Comment:
Please don't create a class just to hold static methods; if there's no state, and it's never initialized, then there's no meaningful class. C functions are a fully supported part of Objective-C, so unlike Java this kind of container is not necessary.